var/spack/repos/builtin/packages/py-grpcio/package.py

```diff
@@ -60,8 +62,7 @@ def setup_build_environment(self, env):
             env.prepend_path('CPATH', query.headers.directories[0])
 
     def patch(self):
-        if self.spec.satisfies('%fj'):
-            filter_file("-std=gnu99", "", "setup.py")
+        filter_file("-std=gnu99", "", "setup.py")
```

Comment:
Build log contains a bunch of warnings like:
```
  cc1plus: warning: command-line option '-std=gnu99' is valid for C/ObjC but not for C++
```
Although this doesn't break the build for GCC, we can still remove it to avoid these warnings.